2017-03-26 23:31:33 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "PLAY_EVOLUTIONS" not found; SQL statement:
select id, hash, apply_script, revert_script, state, last_problem from play_evolutions where state like 'applying_%' [42102-188]
2017-03-27 18:22:14 jdbc[767]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-27 19:20:00 jdbc[361]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-27 20:33:50 jdbc[884]: java.lang.Exception: Open Stack Trace
	at org.h2.util.CloseWatcher.register(CloseWatcher.java:100)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:119)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:735)
	at org.h2.server.web.WebApp.login(WebApp.java:955)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:133)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)

org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-27 20:38:42 jdbc[947]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-27 20:38:42 jdbc[947]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-28 21:09:38 jdbc[1189]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-28 21:09:38 jdbc[1189]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-28 22:09:34 jdbc[1585]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-29 00:40:52 jdbc[677]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-03-29 00:40:52 jdbc[677]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-04-22 16:21:17 jdbc[428]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE ""comment"" (""id"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,""detail"" VARCHAR NOT NULL,""userID"" VARCHAR NOT NULL,""projectID"" VARCHAR NOT NULL))[*] "; SQL statement:
create table "comment" ("id" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,"detail" VARCHAR NOT NULL,"userID" VARCHAR NOT NULL,"projectID" VARCHAR NOT NULL)) [42000-188]
2017-06-13 14:52:24 jdbc[644]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-06-20 17:01:52 jdbc[329]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
drop table "comment" [42102-188]
2017-06-20 17:01:54 jdbc[329]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE ""comment"" (""id"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,""detail"" VARCHAR NOT NULL,""userID"" VARCHAR NOT NULL,""projectID"" VARCHAR NOT NULL))[*] "; SQL statement:
create table "comment" ("id" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,"detail" VARCHAR NOT NULL,"userID" VARCHAR NOT NULL,"projectID" VARCHAR NOT NULL)) [42000-188]
2017-06-20 17:05:12 jdbc[393]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-06-23 15:20:16 jdbc[489]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-06-23 16:36:18 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
drop table "comment" [42102-188]
2017-06-23 16:36:20 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE ""comment"" (""id"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,""detail"" VARCHAR NOT NULL,""userID"" VARCHAR NOT NULL,""projectID"" VARCHAR NOT NULL))[*] "; SQL statement:
create table "comment" ("id" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,"detail" VARCHAR NOT NULL,"userID" VARCHAR NOT NULL,"projectID" VARCHAR NOT NULL)) [42000-188]
2017-06-24 16:19:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
insert into "comment" ("detail","userID","projectID")  values (?,?,?) [42102-188]
2017-06-24 16:24:24 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
insert into "comment" ("detail","userID","projectID")  values (?,?,?) [42102-188]
2017-06-24 16:25:16 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
insert into "comment" ("detail","userID","projectID")  values (?,?,?) [42102-188]
2017-06-24 16:25:49 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
insert into "comment" ("detail","userID","projectID")  values (?,?,?) [42102-188]
2017-06-24 16:27:50 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
insert into "comment" ("detail","userID","projectID")  values (?,?,?) [42102-188]
2017-06-24 16:31:25 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
insert into "comment" ("detail","userID","projectID")  values (?,?,?) [42102-188]
2017-06-24 16:33:20 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Table "comment" not found; SQL statement:
drop table "comment" [42102-188]
2017-06-24 16:33:42 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "CREATE TABLE ""commentProject"" (""id"" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,""detail"" VARCHAR NOT NULL,""userID"" VARCHAR NOT NULL,""projectID"" VARCHAR NOT NULL))[*] "; SQL statement:
create table "commentProject" ("id" BIGINT GENERATED BY DEFAULT AS IDENTITY(START WITH 1) NOT NULL PRIMARY KEY,"detail" VARCHAR NOT NULL,"userID" VARCHAR NOT NULL,"projectID" VARCHAR NOT NULL)) [42000-188]
2017-06-24 16:36:16 jdbc[47]: exception
org.h2.jdbc.JdbcSQLException: Table "commentProject" not found; SQL statement:
drop table "commentProject" [42102-188]
2017-06-24 17:07:24 jdbc[357]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-06-24 17:07:24 jdbc[357]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-06-27 15:37:02 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
WHERE topic like '%p%' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 15:37:08 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
WHERE topic like '%a%' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 15:37:26 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
WHERE TOPIC like '%a%' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 15:37:42 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
WHERE "TOPIC" like '%a%' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 15:37:58 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Table "DETAIL" not found; SQL statement:
SELECT * FROM detail
WHERE "TOPIC" like '%a%' [42102-188]
2017-06-27 15:38:04 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Table "DETAIL" not found; SQL statement:
SELECT  FROM detail
WHERE "TOPIC" like '%a%' [42102-188]
2017-06-27 15:38:11 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Table "DETAIL" not found; SQL statement:
SELECT  TOPIC FROM detail
WHERE "TOPIC" like '%a%' [42102-188]
2017-06-27 15:39:04 jdbc[940]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT "topic" FROM "detail" WHERE topic like '%a%' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:10:20 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where topic like '%a%' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:10:37 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail""
WHERE TOPIC LIKE %[*]A% "; expected "SELECT, FROM"; SQL statement:
SELECT * FROM "detail"
where topic like %a% [42001-188]
2017-06-27 16:12:04 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where topic [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:12:31 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where topic like "%a%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:12:43 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where "TOPIC" like "%a%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:12:59 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT topic FROM "detail"
where "TOPIC" like "%a%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:835)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:13:20 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where "TOPIC" like "%a%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:16:03 jdbc[1002]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:173)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:118)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:91)
	at org.h2.Driver.connect(Driver.java:72)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:92)
	at com.zaxxer.hikari.util.DriverDataSource.getConnection(DriverDataSource.java:98)
	at com.zaxxer.hikari.pool.BaseHikariPool.addConnection(BaseHikariPool.java:438)
	at com.zaxxer.hikari.pool.BaseHikariPool$1.run(BaseHikariPool.java:413)
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	... 16 more
2017-06-27 16:16:37 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where "TOPIC" = 'a' [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:16:50 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail"
where "TOPIC" = "a" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:27:00 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Table "DETAIL" not found; SQL statement:
SELECT * FROM detail [42102-188]
2017-06-27 16:30:01 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "X.TOPIC" not found; SQL statement:
SELECT * FROM "detail" x
where x.topic = "a" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:34:52 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "X.TOPIC" not found; SQL statement:
SELECT * FROM  "detail"  x
where x.topic = "a" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:35:12 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "X.TOPIC" not found; SQL statement:
SELECT * FROM   "detail" as x
where x.topic = "a" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:37:15 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail" 
WHERE topic LIKE "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:37:27 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail"" 
WHERE TOPIC %LIKE[*] ""aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"" "; expected "SELECT, FROM"; SQL statement:
SELECT * FROM "detail" 
WHERE topic %LIKE "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42001-188]
2017-06-27 16:37:33 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail" 
WHERE topic = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:37:50 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail" 
WHERE topic = "a" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:178)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:38:10 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail" 
WHERE topic LIKE "a" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:87)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:38:58 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail"" 
WHRER USERID[*] = ""d98028e4-81b0-4936-93c0-10fda32b023c"" "; SQL statement:
SELECT * FROM "detail" 
WHRER userID = "d98028e4-81b0-4936-93c0-10fda32b023c" [42000-188]
2017-06-27 16:39:09 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail"" 
WHRER USERID[*] =  D98028E4-81B0-4936-93C0-10FDA32B023C "; SQL statement:
SELECT * FROM "detail" 
WHRER userID =  d98028e4-81b0-4936-93c0-10fda32b023c [42000-188]
2017-06-27 16:39:20 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail"" 
WHRER USERID[*]  LIKE  D98028E4-81B0-4936-93C0-10FDA32B023C "; SQL statement:
SELECT * FROM "detail" 
WHRER userID  LIKE  d98028e4-81b0-4936-93c0-10fda32b023c [42000-188]
2017-06-27 16:40:04 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail"" 
WHRER USERID[*]  LIKE  D98028E4-81B0-4936-93C0-10FDA32B023C "; SQL statement:
SELECT * FROM "detail" 
WHRER userID  LIKE  d98028e4-81b0-4936-93c0-10fda32b023c [42000-188]
2017-06-27 16:40:13 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  LIKE  D98028E4-81B0-4936-93C0-10FDA32B023C "; expected "WITH, {"; SQL statement:
WHRER userID  LIKE  d98028e4-81b0-4936-93c0-10fda32b023c [42001-188]
2017-06-27 16:40:56 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  %LIKE  D98028E4-81B0-4936-93C0-10FDA32B023C "; expected "WITH, {"; SQL statement:
WHRER userID  %LIKE  d98028e4-81b0-4936-93c0-10fda32b023c [42001-188]
2017-06-27 16:41:22 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  =  D98028E4-81B0-4936-93C0-10FDA32B023C "; expected "WITH, {"; SQL statement:
WHRER userID  =  d98028e4-81b0-4936-93c0-10fda32b023c [42001-188]
2017-06-27 16:41:35 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  =  ""d98028e4-81b0-4936-93c0-10fda32b023c"" "; expected "WITH, {"; SQL statement:
WHRER userID  =  "d98028e4-81b0-4936-93c0-10fda32b023c" [42001-188]
2017-06-27 16:41:51 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER userID  =  [*]""d98028e4-81b0-4936-93c0-10fda32b023c ;"; SQL statement:
WHRER userID  =  "d98028e4-81b0-4936-93c0-10fda32b023c ; [42000-188]
2017-06-27 16:42:41 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  %LIKE  ""d98028e4-81b0-4936-93c0-10fda32b023c"" "; expected "WITH, {"; SQL statement:
WHRER userID  %LIKE  "d98028e4-81b0-4936-93c0-10fda32b023c" [42001-188]
2017-06-27 16:43:08 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  LIKE  ""d98028e4-81b0-4936-93c0-10fda32b023c%"" "; expected "WITH, {"; SQL statement:
WHRER userID  LIKE  "d98028e4-81b0-4936-93c0-10fda32b023c%" [42001-188]
2017-06-27 16:43:21 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  LIKE  ""%d98028e4-81b0-4936-93c0-10fda32b023c%"" "; expected "WITH, {"; SQL statement:
WHRER userID  LIKE  "%d98028e4-81b0-4936-93c0-10fda32b023c%" [42001-188]
2017-06-27 16:43:39 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "WHRER[*] USERID  LIKE  %D98028E4-81B0-4936-93C0-10FDA32B023C% "; expected "WITH, {"; SQL statement:
WHRER userID  LIKE  %d98028e4-81b0-4936-93c0-10fda32b023c% [42001-188]
2017-06-27 16:45:33 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "NAME" not found; SQL statement:
SELECT * FROM "detail" ORDER BY NAME [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:835)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:45:50 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail" ORDER BY topic [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:835)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:46:33 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "TOPIC" not found; SQL statement:
SELECT * FROM "detail" ORDER BY topic [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.command.dml.Select.prepare(Select.java:835)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:47:41 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic"  LIKE aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:88)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:47:55 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic"  LIKE "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:88)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:48:09 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa%" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic"  LIKE  "%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:88)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:48:21 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa%" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic"  =  "%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:180)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:48:46 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic"  =  "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:180)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:49:21 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "SELECT * FROM ""detail"" ,
WHERE[*] ""topic""    ""aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"""; expected "identifier"; SQL statement:
SELECT * FROM "detail" ,
WHERE "topic"    "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42001-188]
2017-06-27 16:50:45 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic" LIKE "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:88)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:51:15 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa%" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic" LIKE "%aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa%" [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.CompareLike.optimize(CompareLike.java:88)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
2017-06-27 16:55:30 jdbc[1001]: exception
org.h2.jdbc.JdbcSQLException: Column "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA" not found; SQL statement:
SELECT * FROM "detail" 
WHERE "topic"  =aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa [42122-188]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.expression.ExpressionColumn.optimize(ExpressionColumn.java:147)
	at org.h2.expression.Comparison.optimize(Comparison.java:180)
	at org.h2.command.dml.Select.prepare(Select.java:838)
	at org.h2.command.Parser.prepareCommand(Parser.java:246)
	at org.h2.engine.Session.prepareLocal(Session.java:461)
	at org.h2.engine.Session.prepareCommand(Session.java:403)
	at org.h2.jdbc.JdbcConnection.prepareCommand(JdbcConnection.java:1188)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:170)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.h2.server.web.WebApp.getResult(WebApp.java:1388)
	at org.h2.server.web.WebApp.query(WebApp.java:1061)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1023)
	at org.h2.server.web.WebApp$1.next(WebApp.java:1010)
	at org.h2.server.web.WebThread.process(WebThread.java:164)
	at org.h2.server.web.WebThread.run(WebThread.java:89)
	at java.lang.Thread.run(Unknown Source)
